cmd/virt-launcher/virt-launcher.go

```diff
@@ -43,6 +43,10 @@ func markReady(readinessFile string) {
 }
 
 func createSocket(socketDir string, namespace string, name string) net.Listener {
+	socketPath := isolation.SocketFromNamespaceName(socketDir, namespace, name)
+	if err := os.RemoveAll(socketPath); err != nil {
+		log.Fatal("Could not clean up old socket for cgroup detection", err)
```

Comment:
Should this really be a fatal issue? Could "no such file or directory" be a plausible condition here?

Reply:
Is there a way to recover from this error?  Virt-launcher has to be able to create the socket.

Reply:
os.RemoveAll() returns nil if the file/path does not exist.